internal/armada/configuration/types.go

```diff
@@ -187,6 +191,33 @@ type SchedulingConfig struct {
 	AlwaysAttemptScheduling bool
 }
 
+// FairnessType controls how fairness is computed.
+// In particular, each queue has a cost associated with it and the next job to attempt to schedule
+// is taken from the queue with the smallest cost associated with it.
+//
+// Can be either AssetFairness or DominantResourceFairness.
+//
+// If AssetFairness, the cost associated with a queue is a linear combination of its total allocation.
+// E.g., w_CPU * "CPU allocation" + w_memory * "memory allocation".
+//
+// If DominantResourceFairness, the cost associated with a queue is
+// max("CPU allocation" / "CPU capacity", "memory allocation" / "mamory capacity", ...).
```

Comment:
Same comment as above; also, I'd attach the explanations to their respective values.

Reply:
Done